boards/holybro/durandal-v1/nsh/defconfig

```diff
@@ -0,0 +1,256 @@
+#
```

Comment:
Accidental copy of `boards/holybro/durandal-v1/nuttx-config/nsh/defconfig`?

Reply:
yep - I thought I deleted it - we need to check the old config/menu config round trip on bootloader and default.

Reply:
deleted.